packages/paste-website/src/pages/components/table/index.mdx

```diff
@@ -287,58 +343,662 @@ Table rows (`Tr`) can be set with either top, middle, or bottom vertical alignme
+placing a user's email address beneath their username in a single cell under a column header titled "user".
+
+Description text is recommended in cases where the user would benefit from having additional context about a
+particular cell, but that additional context is not the primary data that is being presented in the table.
+For example, in a table that lists a user's email campaigns with open rates, click rates, etc. for each,
+it may help the user identify a particular campaign by knowing both the campaign name and what time it was
+sent, so the first column of this table may include the campaign name with the send time included as
+description text. In this example, the send time is not being compared across rows like the open rates and
+click rates are; instead, the send time is used to add additional detail that helps the user identify and
+understand data in an individual row.
+
+<LivePreview scope={{Table, THead, TBody, Tr, Th, Td, Text}} language="jsx">
+  {`<Table>
+  <THead>
+    <Tr>
+      <Th>User</Th>
+      <Th>Skills</Th>
+      <Th textAlign="right">Tasks</Th>
+    </Tr>
+  </THead>
+  <TBody>
+    <Tr>
+      <Th scope="row">
+        <Text as="p">
+          Adam Brown
+        </Text>
+        <Text as="p" color="colorTextWeak" fontWeight="fontWeightNormal">adam@brown.com</Text>
+      </Th>
+      <Td>English, French, Sales, Spanish</Td>
+      <Td textAlign="right">
+        <Text as="span" fontFamily="fontFamilyCode">35</Text>
+      </Td>
+    </Tr>
+    <Tr>
+      <Th scope="row">
+        <Text as="p" fontWeight="fontWeightBold">
+          Adriana Lovelock
+        </Text>
+        <Text as="p" color="colorTextWeak" fontWeight="fontWeightNormal">adriana@lovelock.com</Text>
+      </Th>
+      <Td>English, French, Sales, Spanish</Td>
+      <Td textAlign="right">
+        <Text as="span" fontFamily="fontFamilyCode">28</Text>
+      </Td>
+    </Tr>
+    <Tr>
+      <Th scope="row">
+        <Text as="p">
+          Amanda Cutlack
+        </Text>
+        <Text as="p" color="colorTextWeak" fontWeight="fontWeightNormal">amanda@cutlack.com</Text>
+      </Th>
+      <Td>English, French, Sales, Spanish</Td>
+      <Td textAlign="right">
+        <Text as="span" fontFamily="fontFamilyCode">7</Text>
+      </Td>
+    </Tr>
+  </TBody>
+</Table>`}
+</LivePreview>
+
+### Table with avatars
+
+Tables can be composed using Avatar, which can be used to make a table of user data more scannable.
+You can put an avatar next to a user's first and last name using `Stack` inside a single `Td`.
+
+<LivePreview scope={{Table, THead, TBody, Tr, Th, Td, Text, Avatar, Box, Stack}} language="jsx">
+  {`<Table>
+  <THead>
+    <Tr>
+      <Th>User</Th>
+      <Th>Skills</Th>
+      <Th textAlign="right">Tasks</Th>
+    </Tr>
+  </THead>
+  <TBody>
+    <Tr>
+      <Th scope="row">
+        <Stack orientation="horizontal" spacing="space30">
```

Comment:
@richbachman can these be `space40` instead of `space30`?

Reply:
and `sizeIcon70` for the avatar size?

Reply:
Yep. I'll make that change.